code/modules/paperwork/pen.dm

```diff
@@ -182,3 +182,22 @@
 	else
 		icon_state = initial(icon_state) //looks like a normal pen when off.
 		item_state = initial(item_state)
+
+/obj/item/proc/on_write(obj/item/weapon/paper/P, mob/user)
+	return
+
+/obj/item/weapon/pen/poison
+	var/uses_left = 3
+
+/obj/item/weapon/pen/poison/on_write(obj/item/weapon/paper/P, mob/user)
+	if(P.contact_poison_volume)
+		to_chat(user, "<span class='warning'>[P] is already coated.</span>")
+	else if(uses_left)
+		uses_left--
+		P.contact_poison = "sarin"
+		P.contact_poison_volume = 15
```

Comment:
This being Sarin kinda negates the point of it being a stealth poison; this isn't what it is on TG.

This is going to get meta'd faster than anything else, because the effects are Sarin are instant.

I'm not a big fan of TG's poison either, but amanitin fits the bill here. I'd make if 14 units; TG's delayed poison does a max of 125 damage; having amanitin in 14 units means it'll do 70 minimum and 140 maximum.